flume/README.md

```diff
@@ -251,14 +251,40 @@ Simply configure the FIWARE repository if not yet configured and use your applic
 Please, refer to [this](doc/installation/src_install.md) document if your aim is to install Cygnus from sources.
 
 ## Cygnus configuration
+Cygnus is configured through two different files:
 
-The typical configuration when using the `HTTPSource`, the `OrionRestHandler`, the `MemoryChannel` and the sinks is shown below (the file `cygnus.conf` can be instantiated from a template given in the clone Cygnus repository, `conf/cygnus.conf.template`).
+* A `cygnus_instance_<id>.conf` file addressing all those non Flume parameters, such as the Flume agent name, the specific log file for this instance, the administration port, etc. This configuration file is not necessary if Cygnus is run as a standlalone application (see later), bt it is mandatory if run as a service (see later).
+* An `agent_<id>.conf` file addressing all those Flume parameters, i.e. how to configure the different sources, channels, sinks, etc. that compose the Flume agent behind the Cygnus instance. always mandatory.
 
-More advanced configurations can be found at [`doc/operation/performance_tuning_tips.md`](doc/operation/performance_tuning_tips.md).
+Please observe there may exist several instances identified by `<id>`, which must be the same for both configuration files regarding the same instance.
```

Comment:
Instances is a bit ambiguous here... it may refer to instances of the files, but without a clear relation with Cygnus instances (as I understand it works). Thus, I'd suggest to specifically mention "Cygnus instances" here.